test/tools/router_check/test/route_tests.sh

```diff
@@ -16,6 +16,13 @@ do
   TEST_OUTPUT=$("${PATH_BIN}" "${PATH_CONFIG}/${t}.yaml" "${PATH_CONFIG}/${t}.golden.json" "--details")
 done
 
+# Testing expected matches using --useproto
+# --useproto needs the test schema as a validation.proto message.
+for t in "${TESTS[@]}"
+do
+  TEST_OUTPUT=$("${PATH_BIN}" "${PATH_CONFIG}/${t}.yaml" "${PATH_CONFIG}/${t}.golden.proto.json" "--details" "--useproto")
```

Comment:
Would it be possible to also have a test case with a golden.proto file?

Reply:
@hennna I did not fully understand the ask. We have already added the golden.proto.json test cases.

Reply:
I was wondering if it is possible now for the second input file to be proto rather than json? Something like:

tests {
  test_name: "Test_1"
   ...
}
tests {
  test_name: "Test_2"
   ...
}

Reply:
cool. I will try that out